client/landing/jetpack-cloud/sections/backups/main.jsx

```diff
@@ -24,64 +25,80 @@ import QuerySitePurchases from 'components/data/query-site-purchases';
 class BackupsPage extends Component {
 	constructor( props ) {
 		super( props );
+
 		this.state = {
```

Comment:
since this is no longer dependent on `moment` it can be moved from the constructer to directly in the class i.e.
```
state = {
   selectedDate: new Date(),
};
```

and we can get rid of the constructor 🎉

Reply:
Yeah! In the other PR, https://github.com/Automattic/wp-calypso/pull/40189, I had already moved it, mainly because my IDE wasn't getting the state for the autocompletion.